packages/core/src/asciidoctor-core-api.js

```diff
@@ -1857,6 +1857,41 @@ Document.prototype.hasHeader = function () {
   return this['$header?']()
 }
 
+/**
+ * Replay attribute assignments at the block level
+ *
+ * @param {Object} blockAttributes - A JSON of attributes
+ *
+ * @memberof Document
+ */
+Document.prototype.playbackAttributes = function (blockAttributes) {
+  blockAttributes = toHash(blockAttributes)
+  if (blockAttributes) {
+    var attrEntries = blockAttributes['$[]']('attribute_entries')
```

Comment:
Why not const?  Same for just about everything here.

Reply:
I'm being conservative, `const` and `let` are not supported in IE < 11 but arguably nobody should use IE < 11.
I will probably drop `var` in a major release so it does not come as a surprise in a feature version.